tests/integration/page/page_enhance_within.js

```diff
@@ -0,0 +1,21 @@
+asyncTest( "Option enhanceWithin", function() {
+	$.testHelper.pageSequence([
+		function() {
+			$( "#open-enhance-within" ).click();
+		},
+		function() {
+			deepEqual( $( "#enhance-within" ).find( ":mobile-toolbar" ).length, 1,
```

Comment:
I don't understand. I'm checking to make sure a toolbar widget was instantiated inside the page with ID ```enhance-within```. How else would I do that?